src/modules/fancyzones/editor/FancyZonesEditor/GridData.cs

```diff
@@ -1025,26 +1025,13 @@ private void FixAccuracyError(List<RowColInfo> info, List<int> percents)
                 total += info[i].Percent;
             }
 
-            int totalDiff = total - 10000;
-            if (totalDiff != 0)
+            int prefixTotal = 0;
+            for (int i = 0; i < percents.Count; i++)
             {
-                int perLineDiff = totalDiff / percents.Count;
-                int lastLineDiff = totalDiff - (perLineDiff * (percents.Count - 1));
-
-                for (int i = 0; i < percents.Count - 1 && perLineDiff != 0; i++)
-                {
-                    int percent = percents[i] - perLineDiff;
-                    if (percent < 0)
-                    {
-                        percent = 0;
-                    }
-
-                    percents[i] = percent;
-                    info[i].Percent = percent;
-                }
-
-                info[percents.Count - 1].Percent -= lastLineDiff;
-                percents[percents.Count - 1] -= lastLineDiff;
+                int first = prefixTotal * 10000 / total;
```

Comment:
In this file there are two `FixAccuracyError` methods, and they both use the hard-coded 10000 value.
It would be better to replace it with a constant and add a comment explaining the reasoning behind this value.